packages/rx_bloc_test/pubspec.yaml

```diff
@@ -1,22 +1,22 @@
 name: rx_bloc_test
 description: A Flutter package with the goal to enable testing rxBlocs from the FlutterRxBloc package with ease.
-version: 3.3.3
+version: 3.4.0
```

Comment:
Increment `major` version number instead of the `minor` one. Because this package is requiring Dart to be 3.0+.